gef.py

```diff
@@ -7616,7 +7618,8 @@ def _hexdump(self, start_addr, length, arrange_as, offset=0):
             sym = "<{:s}+{:04x}>".format(*sym) if sym else ''
             mem = read_memory(cur_addr, l)
             val = struct.unpack(fmt_pack, mem)[0]
-            lines.append(fmt_str % (cur_addr, (i + offset) * l,  sym, val))
+            lines.append(fmt_str.format(base=Color.colorify("{:#x}".format(cur_addr), attrs=base_address_color),
+                                        offset=i * l, sym=sym, val=val))
```

Comment:
This is missing the offset from the last PR:

`offset=(i + offset) * l`

Reply:
Oh good eye, I had some commits mixed up when I rebased.